### ISSUE TITLE
Plugin with id 'maven' not found

### ISSUE THREAD

**Issue by Mohamed-kassim** [Drazail/react-native-hash#40]
**Describe the bug**
After upgrading project to react-native 67.2 it upgrades gradle version to 7 which introduces new changes that fails the build 

new errors while building:
`> Plugin with id 'maven' not found.`

` com.android.builder.errors.EvalIssueException: compileSdkVersion is not specified. Please add it to build.gradle`
 

**To Reproduce**
1. upgrade react-native to 67.2 which upgrades gradle to 7
2. run android

 - OS: Android
 - OS Version: 11
 - React Native Version 67.2

**Smartphone (please complete the following information):**
 - Device: []
 - OS: [Android

**Comment by TruffelNL** [comment]
I don't know if you need an answer or already fixed it yourself, but here is how I fixed it:

In node_modules/react-native-hash/android/build.gradle on line 25 and 48:
change `apply plugin: 'maven'` to `apply plugin: 'maven-publish'`

Same file on line 159:
change `repositories.mavenDeployer {` to `repositories.maven {`

Same file on line 161:
change `repository url: "file://${projectDir}/../android/maven"` to `url = uri("file://${projectDir}/../android/maven")`

Same file line 162:
delete line or comment out (this is not a permanent fix, but I believe it is not needed in end-builds).

That is what made it work again for me. I hope I didn't forget any steps.

**Comment by Drazail** [comment]
@TruffelNL 

Would you be willing to make a pull request?

**Comment by TruffelNL** [comment]
> @TruffelNL
> 
> Would you be willing to make a pull request?

Definitely. The only thing I need to dig into a bit is line 162. I will make a PR as soon as possible.

**Comment by Drazail** [comment]
@Mohamed-kassim please test with master branch of this repo and let us know if your issue is resolved.

**Comment by ca057** [comment]
I ran into the same issue and the build is working again when installing the module from the master branch. Haven’t tested all functionality yet though.

**Comment by Drazail** [comment]
nice, thank you.

will close this issue after the npm package is updated.

**Comment by Drazail** [comment]
[2.0.1 ](https://github.com/Drazail/react-native-hash/releases/tag/2.0.1)release should fix this.

**Comment by anija** [comment]
I was experiencing the same after upgrading gradle, so I updated to v2.0.1 and the previous error was fixed, but now i got 
```
A problem occurred configuring project ':react-native-hash'.
> Configuration with name 'compile' not found.
```
Anyone else with the same problem on this version?

**Comment by Drazail** [comment]
@anija did you manage to fix this?

**Comment by Drazail** [comment]
the problem seems to be originating from miss-matched `gradle` versions, should be fixed in release 2.0.3.
